src/isomorphic/modern/element/__tests__/ReactJSXElementValidator-test.js

```diff
@@ -99,19 +100,49 @@ describe('ReactJSXElementValidator', () => {
       },
     };
 
-    void <Component>{iterable}</Component>;
+    ReactTestUtils.renderIntoDocument(<Component>{iterable}</Component>);
 
     expectDev(console.error.calls.count()).toBe(1);
     expectDev(console.error.calls.argsFor(0)[0]).toContain(
       'Each child in an array or iterator should have a unique "key" prop.',
     );
   });
 
+  it('does not warns for fragments of multiple elements without keys', () => {
```

Comment:
Typo: warn

Reply:
😆 was matching the other typos in this file. I'll fix the typo for all the tests.